src/pdf/ShippingLabels/ShippingLabels.tsx

```diff
@@ -4,6 +4,9 @@ import { Schema } from "@/database_utils";
 import PdfButton from "@/components/PdfButton/PdfButton";
 import ShippingLabelsPdf, { ParcelClients } from "@/pdf/ShippingLabels/ShippingLabelsPdf";
 
+type ParcelWithCollectionCentre = Omit<Schema["parcels"], "collection_centre"> & {
```

Comment:
personal preference: take `collection_centre` out as an interface and then use it in the `ParcelWithCollectionCentre` type

Reply:
Done